app/models/document_type.rb

```diff
@@ -36,6 +36,10 @@ def managed_elsewhere_url
     Plek.new.external_url_for(managed_elsewhere.fetch("hostname")) + managed_elsewhere.fetch("path")
   end
 
+  def parent
+    Supertype.all.find { |supertype| supertype.document_types.map(&:id).include?(id) }
```

Comment:
would `Supertype.all.find { |supertype| supertype.document_types.include?(self) }` work? If not I'd presume we might want to override equality in document_type so it does - though is this stuff all about to change with the work you and Leena are doing?

Reply:
I'm excited about overriding equality - not done that before! 🎉 
Yes it's tricky to do this work in parallel with the multi-level document type work. I figured as this test iteration work is smallest, it will probably be completed & merged first; I can then rebase the other branch. But I'm equally happy parking this for now and then tacking this i18n coverage work on at the end of the other branch.